harmonica/forward/point_mass.py

```diff
@@ -42,22 +37,32 @@ def point_mass_gravity(
 
         l = \sqrt{ (x - x_p)^2 + (y - y_p)^2 + (z - z_p)^2 }.
 
-    The gradient of the potential, also known as the gravity acceleration vector
+    The gradient of the potential, also known as the gravitational acceleration vector
     :math:`\vec{g}`, is defined as:
 
     .. math::
 
-        \vec{g} = \nabla V.
+        \vec{g} = \nabla V
 
-    Therefore, the :math:`z` component of :math:`\vec{g}` at the point :math:`P` can be
-    computed as (remember that :math:`z` points upwards):
+    and has components :math:`g_{northing}(P)`, :math:`g_{easting}(P)` and
+    :math:`g_{upward}(P)` given by
```

Comment:
We could try using LaTeX's `\text` in order to make the subscripts look like text rather than variables.

Reply:
I don't know how to use LaTeX's here

Reply:
You are using LaTeX math mode actually. When using the `:math:`, Sphinx will interpret what's inside as LaTeX math. I was saying to use the `\text` macro like this:
```
:math:`g_\text{northing}(p)`, :math:`g_\text{easting}(p)` and :math:`g_\text{upward}(p)`
```

Reply:
Ok! I can do this

Reply:
I have used `\text` for writing the subscripts of g_northing, g_easting and g_upward components